CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Create a changelog entry for every new user-facing change (where the "users" are plugin developers).
+Please respect the following instructions:
+- Group changes by date, with newest changes first.
+- Prefix your changes with either [Bugfix], [Improvement], [Feature], [Security], [Deprecation].
+- You may optionally append "(by @<author>)" at the end of the line, where "<author>" is either one (just one)
+  of your GitHub username, real name or affiliated organization. -->
+
+# Changelog
+
+## 2023-11-06
+
+- [Improvement] Start keeping a changelog (@kdmccormick).
```

Comment:
```suggestion
- [Improvement] Recommend putting job scripts under "tasks" directory instead of "jobs", for consistency with official plugin (by @CodeWithEmad).
- [Improvement] Start keeping a changelog (by @kdmccormick).
```